<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculMental</class>
 <widget class="QWidget" name="CalculMental">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculMental</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="GroupDisplay">
       <property name="title">
        <string>Display</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="LabNum1">
          <property name="text">
           <string>NUMBER1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LabOpe">
          <property name="text">
           <string>OPE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LabNum2">
          <property name="text">
           <string>NUMBER2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupAns">
       <property name="title">
        <string>Answer</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LabEqual">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TextInput">
          <property name="text">
           <string>ANSWER</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="ButtonResetScore">
          <property name="text">
           <string>Reset Score</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonSubmit">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonNewOpe">
          <property name="text">
           <string>Get New Operation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupDisplay_2">
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="LabInfo">
          <property name="text">
           <string>Res attendu :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LabResAtt">
          <property name="text">
           <string>RES</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LablnfoScore">
          <property name="text">
           <string>SCORE :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LabScore">
          <property name="text">
           <string>SCORE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
